Support/ceptr/ceptr/qssa_converter.py

```diff
@@ -1621,34 +1621,34 @@ def qssa_coeff_functions(
             # Case 2 !PD, TB
             ctuc = cu.prefactor_units(cc.ureg("kmol/m**3"), 1 - dim)
             # pef = (
-            # reaction.high_rate.pre_exponential_factor * ctuc
+            # reaction.rate.high_rate.pre_exponential_factor * ctuc
             # ).to_base_units()
-            beta = reaction.high_rate.temperature_exponent
+            beta = reaction.rate.high_rate.temperature_exponent
             # ae = (
-            # reaction.high_rate.activation_energy
+            # reaction.rate.high_rate.activation_energy
             # * cc.ureg.joule
             # / cc.ureg.kmol
             # ).to(aeuc)
 
             low_pef = (
-                reaction.low_rate.pre_exponential_factor * ctuc
+                reaction.rate.low_rate.pre_exponential_factor * ctuc
             ).to_base_units()
-            low_beta = reaction.low_rate.temperature_exponent
+            low_beta = reaction.rate.low_rate.temperature_exponent
             low_ae = (
-                reaction.low_rate.activation_energy
+                reaction.rate.low_rate.activation_energy
                 * cc.ureg.joule
                 / cc.ureg.kmol
             ).to(aeuc)
-            if reaction.rate.type == "Troe":
+            if "Troe" in reaction.reaction_type:
                 troe = reaction.rate.falloff_coeffs
                 ntroe = len(troe)
                 is_troe = True
-            elif reaction.rate.type == "Sri":
+            elif "Sri" in reaction.reaction_type:
                 pass
                 # sri = reaction.rate.falloff_coeffs
                 # nsri = len(sri)
                 # is_sri = True
-            elif reaction.rate.type == "Lindemann":
```

Comment:
Good point. Looks like there is now 
```
In [7]: reac.rate
Out[7]: <TroeRate at 1066df870>

In [8]: reac.rate.type
Out[8]: 'falloff'

In [9]: reac.rate.sub_type
Out[9]: 'Troe'
```
so I will switch to using those tomorrow instead of using
```
In [10]: reac.reaction_type
Out[10]: 'falloff-Troe'
```

Reply:
done